pom.xml

```diff
@@ -30,8 +34,8 @@
   <groupId>com.xwiki.ideas</groupId>
   <artifactId>application-ideas</artifactId>
   <version>1.13.2-SNAPSHOT</version>
-  <packaging>xar</packaging>
-  <name>Ideas Application</name>
+  <packaging>pom</packaging>
```

Comment:
Note that you're changing the **extension id** of the Ideas Application: ``application-ideas`` -> ``application-ideas-ui``. This has important implications. We would have to update the extension repository, the XWiki Pro package, all licenses generated for ``application-ideas`` will have to be re-generated, but more importantly, without special handling, the Extension Manager will not perform an upgrade (it has no idea that ``application-ideas-ui`` is the same as ``application-ideas`` unless you tell it so). It would be easier to keep the id ``application-ideas`` for the UI and use ``application-ideas-pom`` for the parent.